webapp/channels/src/components/channel_view/channel_view.tsx

```diff
@@ -171,7 +171,7 @@ export default class ChannelView extends React.PureComponent<Props, State> {
                 <div
                     id='post-create'
                     data-testid='post-create'
-                    className='post-create__container AdvancedTextEditor__ctr'
+                    className='post-create__container'
```

Comment:
I think this that class name is actually needed because of this one line of CSS
https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/advanced_text_editor/advanced_text_editor.scss#L20

Reply:
added back